io.github.Figma_Linux.figma_linux.yml

```diff
@@ -0,0 +1,53 @@
+id: io.github.Figma_Linux.figma_linux
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+base: org.electronjs.Electron2.BaseApp
+base-version: '20.08'
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --share=network
+  - --filesystem=home
+  - --filesystem=xdg-pictures
+command: figma-linux
+rename-icon: figma-linux
+modules:
+
+  - name: figma-linux
+    buildsystem: simple
+    build-commands:
+      - ar x figma-linux_*.deb
+      - rm figma-linux_*.deb
+      - tar xf data.tar.xz 
+      - cp -r  opt/figma-linux /app/figma-linux
+      - mkdir -p /app/share/icons/hicolor
+      - cp -r usr/share/icons/hicolor/* /app/share/icons/hicolor
+      - install figma-linux.sh /app/bin/figma-linux
+      - install -Dm644 io.github.Figma_Linux.figma_linux.desktop /app/share/applications/io.github.Figma_Linux.figma_linux.desktop
+      - install -Dm644 ${FLATPAK_ID}.appdata.xml -t /app/share/appdata/
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+
+        url: https://github.com/Figma-Linux/figma-linux/releases/download/v0.7.2/figma-linux_0.7.2_linux_amd64.deb
+        sha256: e3fef97591d716701f400064ac085308437a259a2395cb74337cccd6ef606dd9
+        size: 76809218
+
+        x-checker-data:
+          type: debian-repo
+          package-name: figma-linux
+          dist: all
+          component: main
```

Comment:
There's no debian-repo here so it won't work. Using JSON checker with GitHub API might be a better idea: https://github.com/flathub/flatpak-external-data-checker/#json-checker

Reply:
> The desktop and appdata files should be ideally upstreamed. Just to double-check, @ChugunovRoman, would you be interested in co-maintaining your app on Flathub?

Ok, I don't mind the package support on Flathub.
Am I correct in understanding that after accepting this PR, a repository will be created and I will just need to update it?

Reply:
Yes, that's exactly it.